diff -urN a/deps/plog/CMakeLists.txt b/deps/plog/CMakeLists.txt
--- a/deps/plog/CMakeLists.txt	2021-08-05 14:55:14.940435259 +0000
+++ b/deps/plog/CMakeLists.txt	2021-08-05 14:57:52.771003626 +0000
@@ -34,9 +34,9 @@
     add_subdirectory(samples)
 endif()
 
-install(
-    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/plog
-    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-    FILES_MATCHING # headers only
-    PATTERN "*.h"
-)
+# install(
+#     DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/plog
+#     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+#     FILES_MATCHING # headers only
+#     PATTERN "*.h"
+# )
