diff -urN a/deps/json/CMakeLists.txt b/deps/json/CMakeLists.txt
--- a/deps/json/CMakeLists.txt	2021-06-02 09:24:15.614185214 +0000
+++ b/deps/json/CMakeLists.txt	2021-06-02 09:21:30.046945216 +0000
@@ -16,7 +16,7 @@
 ## OPTIONS
 ##
 option(JSON_BuildTests "Build the unit tests when BUILD_TESTING is enabled." ON)
-option(JSON_Install "Install CMake targets during install step." ON)
+option(JSON_Install "Install CMake targets during install step." OFF)
 option(JSON_MultipleHeaders "Use non-amalgamated version of the library." OFF)
 
 ##
diff -urN a/deps/plog/CMakeLists.txt b/deps/plog/CMakeLists.txt
--- a/deps/plog/CMakeLists.txt	2021-06-02 09:24:16.424188019 +0000
+++ b/deps/plog/CMakeLists.txt	2021-06-02 09:21:39.286977221 +0000
@@ -34,9 +34,9 @@
     add_subdirectory(samples)
 endif()
 
-install(
-    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/plog
-    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-    FILES_MATCHING # headers only
-    PATTERN "*.h"
-)
+# install(
+#     DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/plog
+#     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+#     FILES_MATCHING # headers only
+#     PATTERN "*.h"
+# )
